### ISSUE TITLE
About the baseline BERT base model

### ISSUE THREAD

**Issue by yxgnahz** [zinengtang/VidLanKD#5]
Hi,
Could you tell us how do you train the baseline BERT_base model in Table 1. If possible, could you release this checkpoint?
Thanks in advance.

**Comment by zinengtang** [comment]
You can try this command. I updated in the README.
bash scripts/base_wiki.bash 0,1,2,3 wiki_bert_base
I'll take a look if I have the checkpoint.

**Comment by yxgnahz** [comment]
Thanks for your reply.
I used this script to pretrain a BERT and I finetuned it on GLUE. Here are the results:
1. Setting lr to 1e-4, batch size to 32, finetuning epoch to 3 (same as the description in the paper)

Method | SST2 | QNLI | QQP | MNLI
-- | -- | -- | -- | --
BERT-12L-vokenization-paper | 89.3 | 87.9 | 83.2 | 79.4
BERT-12L-VidLanKD-paper | 89 | 88 | 86.2 | 79.2
BERT-12L-reimp-seed-1 | 89.2 | 88.1 | 87.5 | 36.6
BERT-12L-reimp-seed-42 | 88.5 | 88.7 | 87.1 | 79.5
BERT-12L-reimp-seed-100 | 89.7 | 87.9 | 87 | 32.7
BERT-12L-reimp-seed-9595 | 89.8 | 88.2 | 59 | 80.5
BERT-12L-reimp-seed-123456 | 91.2 | 87.4 | 86.3 | 80.5

I tried five different random seeds, and I found that the finetuning results are very unstable. Especially for QQP and MNLI, with some random seeds the results can be much worse. This phenomenon is very similar to #2 

2. Then I setting lr to 1e-4, batch size 128(4 GPUs with 32 samples on each), finetuning epoch to 3 (same as the sample in this repo)


Method | SST2 | QNLI | QQP | MNLI
-- | -- | -- | -- | --
BERT-12L-vokenization-paper | 89.3 | 87.9 | 83.2 | 79.4
BERT-12L-VidLanKD-paper | 89 | 88 | 86.2 | 79.2
BERT-12L-reimp-seed-1 | 90.4 | 89.4 | 88.5 | 81.5
BERT-12L-reimp-seed-42 | 90.0 | 89.7 | 88.7 | 81.2
BERT-12L-reimp-seed-100 | 90.1 | 89.2 | 88.5 | 81.3
BERT-12L-reimp-seed-9595 | 89.9 | 88.7 | 88.4 | 81.3
BERT-12L-reimp-seed-123456 | 91.4 | 88.5 | 88.9 | 81.7

As you can see, when setting lr to 128 (per_GPU_batch_size 32 with 4 GPUs), the finetuning is much more stable. Therefore, I wonder in your paper, which setting do you use finally(batch size 32 or 128)? Besides, could you tell me the detailed setting for finetuning SQUAD and SWAG?

**Comment by zinengtang** [comment]
Hi. Did you use any warmup learning rate? I think this will help.
Also, are you using fp16 precision.
To my knowledge these 2 configs matter a little bit for the stability of the training
